afroinnovate/afi_loadboard_api
# Swap sqlserver with postgres and Containerize the APIS

- Containerize the freight api
- Swap sqlserver with Postgres
- Test both apis to be workin.
- ## Test
- Run - dotnet ef database update
- Run - docker-compose -f dev-docker-compose.yaml up --build
- APIS should be built and spun up.
- access them both in 8080 and 7070 respectiviely auth and Freight
![image](https://github.com/afroinnovate/afi_loadboard_api/assets/44912577/6f6412ba-22f0-486d-a0df-9b027372542a)

Auth:
![image](https://github.com/afroinnovate/afi_loadboard_api/assets/44912577/4bf37578-32bd-43f0-92dc-b73195238023)
